### DISCUSSION TITLE
Custom Login Appears for an instant then goes away. Is there a way to keep it?

### DISCUSSION THREAD

**Discussion by MrGonzo1024** [Budibase/budibase#4966] [Help]
Hi,

I've read through the threads that custom login is not supported because Budibase focus is on intranet versus exranet.  But I've look at many many low code development platforms and when I found Budibase, it was perfect for what I needed!! It's a great solution, it beats all the others hands down. The only thing missing was a custom login screen. 

But, I noticed when connecting to my test application through my chrome incognito browser, my Organization Name and Logo appear on the login screen for a brief instant, then it changes to the Budibase Name and Logo.  So, the functionality is there. Is there a way to stop it from switching to the Budibase Name and Logo?  Is that because I'm using the Budibase cloud right now? Would it be different if I self hosted?

Thanks!!

**Comment by mjashanks** [comment]
Hey - this is actually a Bug with Budibase cloud. You should be able to cha get your company name and logo but cannot. It works on self-hosted.

We should get this fixed in the next release.

**Reply by MrGonzo1024** [reply]
Hi.  I set it up self hosted and it does work!! Thanks Mark!
